examples/google.js

```diff
@@ -16,8 +17,24 @@ function showMap(key) {
     }
   });
 
+  class GoogleLogoControl extends Control {
+    constructor() {
+      const element = document.createElement('img');
+      element.style.pointerEvents = 'none';
+      element.style.position = 'absolute';
+      element.style.bottom = '5px';
+      element.style.left = '5px';
+      element.src =
+        'https://developers.google.com/static/maps/documentation/images/google_on_white.png';
+      super({
+        element: element,
+      });
+    }
+  }
+
   const map = new Map({
     layers: [new Layer({source})],
+    controls: defaultControls().extend([new GoogleLogoControl()]),
```

Comment:
I'm not sure, but the terms might require the attribution to be always visible. If that's the case, the controls should be configured appropriately:
```suggestion
    controls: defaultControls({
      attributionOptions: {
        collapsible: false
      }
    }).extend([new GoogleLogoControl()]),
```
What's your interpretation of the terms here?

Reply:
Their doc states
++++
_If it's infeasible to display data attributions in full due to viewport size constraints, consider adding a hover-over or clickable UI element labeled "Data sources", which opens within the map window to provide attribution information._
++++
So you are right i think. When the map is not displayed on a small screen, there is room to display the attibutions, and it should be so. Best would be a "responsive" change of this collapsible property, but it might be a bit far fetched :) i'll commit your changes

Reply:
Would it be better to default `attributionsCollapsible` in the source super similar to OSM?

Reply:
Sure, i'll add it the same way